{"uuid":"d7b22ee3-16c3-40ae-870b-3432353b1e21","historyId":"de628c1165eeee559eee093200b258ea","fullName":"com.trendyol.test.Services.ServicesTest.getNotFoundBookID","labels":[{"name":"package","value":"com.trendyol.test.Services.ServicesTest"},{"name":"testClass","value":"com.trendyol.test.Services.ServicesTest"},{"name":"testMethod","value":"getNotFoundBookID"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"sondeneme"},{"name":"subSuite","value":"com.trendyol.test.Services.ServicesTest"},{"name":"host","value":"ebrukaya-MacBook-Pro.local"},{"name":"thread","value":"20641@ebrukaya-MacBook-Pro.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"getNotFoundBookID","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"açıklamaio.restassured.internal.http.HttpResponseException: status code: 404, reason phrase: Not Found","trace":"java.lang.AssertionError: açıklamaio.restassured.internal.http.HttpResponseException: status code: 404, reason phrase: Not Found\n\tat org.testng.Assert.fail(Assert.java:98)\n\tat com.trendyolbase.utility.Log.fail(Log.java:42)\n\tat com.trendyolbase.utility.Log.fail(Log.java:33)\n\tat com.trendyolbase.basepages.ServicesBase.getRequest(ServicesBase.java:40)\n\tat com.trendyolpages.service.trendyolServices.getSingleBook(trendyolServices.java:27)\n\tat com.trendyol.test.Services.ServicesTest.getNotFoundBookID(ServicesTest.java:42)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.util.ArrayList.forEach(ArrayList.java:1259)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:808)\n\tat org.testng.TestRunner.run(TestRunner.java:603)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:429)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:326)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1169)\n\tat org.testng.TestNG.runSuites(TestNG.java:1092)\n\tat org.testng.TestNG.run(TestNG.java:1060)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\n"},"stage":"finished","steps":[{"name":"Get Not Found Book ID","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"açıklamaio.restassured.internal.http.HttpResponseException: status code: 404, reason phrase: Not Found","trace":"java.lang.AssertionError: açıklamaio.restassured.internal.http.HttpResponseException: status code: 404, reason phrase: Not Found\n\tat org.testng.Assert.fail(Assert.java:98)\n\tat com.trendyolbase.utility.Log.fail(Log.java:42)\n\tat com.trendyolbase.utility.Log.fail(Log.java:33)\n\tat com.trendyolbase.basepages.ServicesBase.getRequest(ServicesBase.java:40)\n\tat com.trendyolpages.service.trendyolServices.getSingleBook(trendyolServices.java:27)\n\tat com.trendyol.test.Services.ServicesTest.getNotFoundBookID(ServicesTest.java:42)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.util.ArrayList.forEach(ArrayList.java:1259)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:808)\n\tat org.testng.TestRunner.run(TestRunner.java:603)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:429)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:326)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1169)\n\tat org.testng.TestNG.runSuites(TestNG.java:1092)\n\tat org.testng.TestNG.run(TestNG.java:1060)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\n"},"stage":"finished","steps":[{"name":"açıklamaio.restassured.internal.http.HttpResponseException: status code: 404, reason phrase: Not Found","status":"failed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1655061600176,"stop":1655061600176}],"attachments":[{"name":"Request","source":"045c9cfc-5680-4965-b3b5-b860000511e6-attachment.html","type":"text/html"}],"parameters":[],"start":1655061599344,"stop":1655061600177}],"attachments":[],"parameters":[],"start":1655061599344,"stop":1655061600177}